src/App.jsx

```diff
@@ -1,104 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+// funciton for take all data comments
+function getCommentbyId(postId) {
```

Comment:
Consider renaming the function `getCommentbyId` to `getCommentsByPostId` for better clarity and consistency with naming conventions.